packages/mui-joy/src/Slider/Slider.tsx

```diff
@@ -1,41 +1,51 @@
-import PropTypes from 'prop-types';
 import * as React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
Can we extract these changes to a different PR? They don't seem related to the changes proposed in this PR

Reply:
Yes, I moved it to this one already. https://github.com/mui/material-ui/pull/33051

Reply:
It is merged! Ready for another round of review.